README.md

```diff
@@ -66,10 +66,30 @@ Task manifest configuration is described in [snap's documentation](https://githu
 - `"port"` -  KairosDB REST API port (ex. `"8083"`)
 - `"useDynamic"` -  Use dynamic namespaces for metric and tags (ex. `true`)
 
+To know more about Dynamic Metric Namespace, please refer to [snap's documentation](https://github.com/intelsdi-x/snap/blob/master/docs/METRICS.md#dynamic-metrics)
+
 See example task manifest in [examples/tasks/] (https://github.com/intelsdi-x/snap-plugin-publisher-kairosdb/blob/master/examples/tasks/).
 
 ### Examples
-Example of running [psutil collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-psutil) and publishing data to KairosDB.
+Here are samples to illustrate the differences for `useDynamic` flag with [plugin-collector-cpu](https://github.com/intelsdi-x/snap-plugin-collector-cpu)
```

Comment:
Could you make this example with [snap-plugin-collector-psutil](https://github.com/intelsdi-x/snap-plugin-collector-psutil)? We will have a consistent example. snap-plugin-collector-psutil has also dynamic metrics and it is easy to use as snap-plugin-collector-cpu.